tokendito/helpers.py

```diff
@@ -602,6 +606,103 @@ def update_aws_config(profile, output, region):
         config.write(file)
 
 
+def check_within_range(user_input, valid_range):
+    """Validate the user input is within the range of the presented menu.
+
+    This should only get invoked when valid_range is not None, which
+    is only true during menu selection validation and not for TOTP validation.
+
+    :param user_input: integer-validated user input.
+    :param valid_range: the valid range presented on the user's menu.
+    :return range_validation: true or false
+    """
+    range_validation = False
+    if int(user_input) in range(0, valid_range):
+        range_validation = True
+    else:
+        logging.debug("Valid range is {}".format(valid_range))
+        logging.error("Value is not in within the selection range.")
+    return range_validation
+
+
+def check_integer(value, valid_range):
+    """Validate integer.
+
+    :param value: value to be validated.
+    :return passcode:
+    """
+    integer_validation = False
+    try:
+        if int(value):
+            integer_validation = True
+    except ValueError:
+        logging.error("Input is not an integer.")
+
+    return integer_validation
```

Comment:
what happens if the input is a float, or in octal / hex representation? Does it matter? We should probably write up a test for this function.